doc/ug/dv_methodology.md

```diff
@@ -20,7 +20,7 @@ For FPV, some DUTs may also include an SV testbench along with design properties
 As the project is still in development, the current status will not be completed for all IP, but that is the ultimate goal.
 See discussion below on tracking progress.
 
-For professional tooling, the team has chosen [Synopsys' VCS](https://www.synopsys.com/verification/simulation/vcs.html) as the simulator of choice with respect to the tracking of verification completeness and [JasperGold](https://www.cadence.com/content/cadence-www/global/en_US/home/tools/system-design-and-verification/formal-and-static-verification/jasper-gold-verification-platform.html) for FPV.
+For professional tooling, the team has chosen [Synopsys' VCS](https://www.synopsys.com/verification/simulation/vcs.html) as the simulator of choice with respect to the tracking of verification completeness. [JasperGold](https://www.cadence.com/content/cadence-www/global/en_US/home/tools/system-design-and-verification/formal-and-static-verification/jasper-gold-verification-platform.html) and [VC Formal] (https://www.synopsys.com/verification/static-and-formal-verification/vc-formal.html) for FPV.
```

Comment:
Hi Will, I think since we are not using VC Formal for sign off, i prefer to leave this paragraph as it is, and change the next paragraph to add VC Formal. As suggested below:
```
At this time there is also some support for Cadence's Xcelium, for those organizations which have few Synopsys VCS licenses.
However support is not as mature as for VCS, which remains the tool for signoff.
Similarly for FPV, there is also some support for [VC Formal](https://www.synopsys.com/verification/static-and-formal-verification/vc-formal.html), but not as matured for the signoff. 
```
Please let me know what you think.